src/Components/NavBar.js

```diff
@@ -0,0 +1,15 @@
+import { NavLink } from "react-router-dom";
+import "./NavBar.css";
+
+const NavBar = () => {
+    return(
+        <nav className="NavBarContainer">
+            <NavLink to={"/"}><img src="https://cdn.dribbble.com/users/3117394/screenshots/6979664/kaonashi_still_2x.gif?compress=1&resize=50x50" alt="no-face"></img></NavLink>
```

Comment:
Add an exact to the / NavLink so the active class isn't triggered on all routes.